boilerplate/lyft/golang_test_targets/download_tooling.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Everything in this file needs to be installed outside of current module
+# The reason we cannot turn off module entirely and install is that we need the replace statement in go.mod
+# because we are installing a mockery fork. Turning it off would result installing the original not the fork.
+# We also want to version all the other tools.  We also want to be able to run go mod tidy without removing the version
+# pins.  To facilitate this, we're maintaining two sets of go.mod/sum files - the second one only for tooling.  This is
+# the same approach that go 1.14 will take as well.
+# See:
+#   https://github.com/lyft/flyte/issues/129
+#   https://github.com/golang/go/issues/30515 for some background context
+#   https://github.com/go-modules-by-example/index/blob/5ec250b4b78114a55001bd7c9cb88f6e07270ea5/010_tools/README.md
+
+set -e
+
+# List of tools to go get
+# In the format of "<cli>:<package>" or ":<package>" if no cli
+tools=(
+  "mockery:github.com/vektra/mockery/cmd/mockery"
+  "pflags:github.com/lyft/flytestdlib/cli/pflags"
+  "golangci-lint:github.com/golangci/golangci-lint/cmd/golangci-lint"
+  "enumer:github.com/alvaroloes/enumer"
+)
+
+tmp_dir=$(mktemp -d -t gotooling-XXX)
+echo "Using temp directory ${tmp_dir}"
+cp -R boilerplate/lyft/golang_support_tools/* $tmp_dir
+pushd "$tmp_dir"
+
+for tool in "${tools[@]}"
+do
+    cli=$(echo "$tool" | cut -d':' -f1)
+    package=$(echo "$tool" | cut -d':' -f2)
+    echo "Installing ${package}"
+    GO111MODULE=on go install "$package"
```

Comment:
So we install it anyway without checking existence? If that is the case we don't need that column hack.

Reply:
yeah, cuz we want to make sure the version is the right one

Reply:
Yeh I see that point totally. We can remove that column cut thing so the code became much simpler. Also I would suggest we enable shellcheck on TravisCI.